src/components/contentful/contentfulExternalForm/contentfulExternalForm.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import './contentfulExternalForm.scss';
+
+const ContentfulExternalForm = ({ content }) => {
+  const { linkText, formUrl, openInNewTab } = content;
+  return (
+    <section className="ExternalForm layout-container">
+      <a
+        className="ExternalForm__link"
+        href={formUrl}
+        {...(openInNewTab ? { target: '_blank' } : {})}
+      >
+        {linkText}
+      </a>
```

Comment:
I would be very cautious of using `target: '_blank'` anywhere. It is changing the default behaviour, and can be really confusing for screen reader users (especially without an indicator of that happening), and can be confusing for sighted people as well. That's also taking user's decision of if they want to open the window in new tab, or the same one, away. Here's [an article from accessibility confomance point](https://adrianroselli.com/2020/02/link-targets-and-3-2-5.html) of this thing.

So, if we want to still do that `target: '_blank'`, there needs to be a way to communicate this to the user with "Opens in new window". It needs to be text, that screen readers can read (so, not in, for example, CSS-::after-pseudoelement), and in a way that sighted users can understand it too. I think it should be done in the code level - as it is so easy to forgot to provide the text about opening a new window when creating content on the Contentful side. 

So, what do you think about these thoughts? 😊

Reply:
Thanks Eevis for sharing the article! : ) I was not aware about `target: '_blank'` before!

So I updated the link component and its styling (since the design was not done by Mari, so it's free to be modified : D) - we just need to go through with Mari afterwards to see if she'd like to make any adjustments.

![image](https://user-images.githubusercontent.com/6004147/112095226-cc18b200-8ba4-11eb-9435-eefec50af849.png)